devices/google-blueline/kernel-mainline/default.nix

```diff
@@ -0,0 +1,48 @@
+{
+  mobile-nixos
+, fetchFromGitHub
+, fetchpatch
+, ...
+}:
+
+mobile-nixos.kernel-builder rec {
+  version = "5.13.0-rc2";
+  configfile = ./config.aarch64;
+
+  #src = /Users/samuel/tmp/linux/linaro-pixel3;
+
+  # FIXME: apply patchsets directly on top of mainline
+  src = fetchFromGitHub {
+    # Mirror of http://git.linaro.org/people/vinod.koul/kernel.git/commit/?h=pixel/dsc_rfc&id=dcf128e83ddd25e461a5080a349de5a0552eed32
+    owner = "samueldr";
+    repo = "linux";
+    rev = "dcf128e83ddd25e461a5080a349de5a0552eed32";
+    sha256 = "07xp9bi0xl54yfyspaw2aa410142xvy38p2l6z92ximajlgxh1gv";
+  };
+
+  patches = [
+    ./0001-HACK-Add-back-TEXT_OFFSET-in-the-built-image.patch
+  ];
+
+  # FIXME: generic mainline build; append per-device...
```

Comment:
I probably want to look at `asus-dumo` and `pine64-pinephone` (and pinetab by extension) at the same time. To figure out a good pattern for a generic mainline builder?